### DISCUSSION TITLE
Create a new admin user from command line

### DISCUSSION THREAD

**Discussion by huaying** [strapi/strapi#7168] [Legacy Topics]
Hi,

I tried to upgrade from 3.0.20.beta to 3.1.1 and I found I have to login with email after migrations.
I don't know what was my email. How can I create a new admin? Can I login with username like before?

Thanks.

**Comment by lauriejim** [comment]
Hello! I suppose you have access to the database.
You can check directly there.

**Comment by huaying** [comment]
Thanks! Another question is that we are now forced to use email to login?

**Reply by lauriejim** [reply]
For the admin, yes you are.
